# src: ubuntu

Description: Include string.h header for strlen prototype for GCC 4.4. compatibility.
Author: Chris Lamb <chris@chris-lamb.co.uk>

diff -urNad /tmp/bp-build/udptunnel-1.1.orig.orig/host2ip.c /tmp/bp-build/udptunnel-1.1.orig/host2ip.c
--- a/host2ip.c	2008-08-28 07:50:41.000000000 +0100
+++ b/host2ip.c	2008-08-28 07:50:49.000000000 +0100
@@ -6,6 +6,7 @@
 #include <arpa/inet.h>       /* inet_addr() */
 #include <rpcsvc/ypclnt.h>   /* YP */
 #include <ctype.h>           /* isspace() */
+#include <string.h>          /* strlen() */
 
 #include "host2ip.h"
 
